recipes/xvega/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DBUILD_TESTS=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
```

Comment:
Also, this should be a release build

Reply:
Even if it is header-only it is a good idea to make this a release build in case of future changes

Reply:
good point.